rdbi-recipes/src/main/java/com/lithium/dbi/rdbi/recipes/channel/ChannelLuaReceiver.java

```diff
@@ -38,6 +41,32 @@ public GetResult get(
         );
 
         @Query(
+            "local tables_size = KEYS[1]\n" +
+            "local bulk_result = {}\n" +
+            "for i = 1, tables_size do\n" +
+            "    local current_count = redis.call(\"GET\", tostring(KEYS[i*3]))\n" +
+            "    if not current_count then\n" +
+            "        current_count = 0\n" +
+            "    else\n" +
+            "        current_count = tonumber(current_count)\n" +
+            "    end\n" +
+            "    if current_count <= tonumber(KEYS[i*3+1]) then\n" +
+            "        bulk_result[i] = tostring(0)\n" +
+            "    else\n" +
+            "        local results = redis.call(\"LRANGE\", KEYS[i*3-1], 0, current_count - tonumber(KEYS[i*3+1]) + 1)\n" +
+            "        results[#results + 1] = tostring(current_count)\n" +
+            "        bulk_result[i] = results\n" +
+            "    end\n" +
+            "end\n" +
+            "return bulk_result;"
+        )
+        @Mapper(GetBulkResultMapper.class)
+        public GetBulkResult getMultiV2(
+                @BindKey("allKeys") List<String> inputKeys,
+                @BindArg("allArgs") List<String> inputArgs
```

Comment:
woah whoa, let's talk, you're suppose to use the args list, the keys list is only to tell redis what keys you'll be touching, the arg list should contain the depths etc.